lib/config/definitions.ts

```diff
@@ -7,6 +7,13 @@ import * as pep440Versioning from '../versioning/pep440';
 import type { RenovateOptions } from './types';
```

Comment:
Could make sense to add a `default` key to the `gitNoVerify` option in this MR, realized that was missing. The default is `['commit', 'push']`.